src/app/campaign-details/campaign-details.component.html

```diff
@@ -5,23 +5,28 @@ <h2>Banner Image here!</h2>
 <div class="details" *ngIf="campaign">
 	<div>
 		<h2>{{campaign.title}}</h2>
-    <h3 *ngIf="campaign.championName">Championed by {{campaign.championName}}</h3>
+		<h3 *ngIf="campaign.championName">Championed by {{campaign.championName}}</h3>
```

Comment:
@tbg-jasonfung this is still changing various spaces to tabs, please check your editor config and the diff for changes already applied